### ISSUE TITLE
Twisted fillet on fused shape

### ISSUE THREAD

**Issue by pyoccMAN** [tpaviot/pythonocc-core#612]
Hello,

I am trying to make a fillet on fused shape.  I have used the SetRadius option but the fillet is twisted:
![image](https://user-images.githubusercontent.com/46593271/51024405-c4533100-1589-11e9-895e-e05e2a3dd510.png)
 I am not sure how SetRadius works. One first specifies radius, and than the contour index, but the third parameter I am not sure.

Thank you.

**Comment by tpaviot** [comment]
Hard to say what's going on without more details.

**Comment by pyoccMAN** [comment]
Alos, some surfaces are missing:
![image](https://user-images.githubusercontent.com/46593271/51032369-0a1af400-15a0-11e9-8dc5-6e883eeaab34.png)

**Comment by pyoccMAN** [comment]
The contour is shown in the picture below:
![image](https://user-images.githubusercontent.com/46593271/51033243-08066480-15a3-11e9-9a1b-49245e626ee8.png)

**Comment by tpaviot** [comment]
```SetRadius``` is a class method. Which class to you use to perform the fille ? BRepFilletAPI_MakeFillet ?

You can have a look at fillet related examples here https://github.com/tpaviot/pythonocc-demos/blob/master/examples/core_topology_fillet.py

**Comment by pyoccMAN** [comment]
Yes, I use BRepFilletAPI_MakeFillet. I had a look at both examples regarding fillets. Add class method fails, and  SetRadius makes the fillets but twisted. I am not sure what is :type IinC: int reffered in the SetRadius. Is there another suitable class for fillets? Can I specify tangent for fillets?

**Comment by tpaviot** [comment]
Documentation related to chamfers and fillets can be read at https://www.opencascade.com/doc/occt-6.9.1/overview/html/occt_user_guides__modeling_algos.html#occt_modalg_6

**Comment by tpaviot** [comment]
And the documentation related to the ```BRepFilletAPI_MakeFillet``` class https://www.opencascade.com/doc/occt-6.9.1/refman/html/class_b_rep_fillet_a_p_i___make_fillet.html

This is the best I can do, please report any progress.
